canonical-web-and-design/microstack.run
# Add run file

## Done

- Add run file to enable using ./run

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8039/
- Check site runs as expected
